src/test/kotlin/com/ampnet/reportserviceeth/service/TransactionsSummaryTest.kt

```diff
@@ -47,6 +47,15 @@ class TransactionsSummaryTest : TestBase() {
         assertThat(txSummary.dateOfFinish).isEqualTo(getDateOfFinish(periodRequest))
     }
 
+    @Test
+    fun testIssuer() {
+        val i = IIssuer.IssuerState(
+            BigInteger.ONE, userAddress, userAddress, userAddress, userAddress,
+            userAddress, userAddress, "QmYuSijGgZAnBadguWUjLTYyfbvpaUBoWRfQMveo6XfzP3"
+        )
+        println(i.toString())
+    }
+
```

Comment:
WTF should this test do, verify?

Reply:
Haha, I forgot to remove it, I had issues initialising the IIssuer.IssuerState object so I used a test.